src/modules/snap/index.ts

```diff
@@ -87,7 +87,7 @@ export async function initiatePolkdatodSnap(): Promise<SnapInitializationRespons
   try {
     console.info('Attempting to connect to snap...');
     // Todo: update network params
-    const metamaskPolkadotSnap = await enablePolkadotSnap({ networkName: 'shibuya' }, snapId);
+    const metamaskPolkadotSnap = await enablePolkadotSnap({ networkName: 'astar' }, snapId);
```

Comment:
@gluneau am I correct that it doesn't matter passing astar while the portal connects to 'Shiden' or 'Shibuya'? Otherwise, you can use networkNameSubstrate from useNetworkInfo hook. You might want to check the actual value under each networks and make it lower case.

Reply:
I've introduced the `networkNameSubstrate` but it seems that the Store or getters not defined yet.

Let me know.